<launch>
    <!-- webcontroller -->
    <include file="$(find roswww)/launch/roswww.launch"/>
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
    <!-- dynamixel-->
    <include file="$(find dynamixel_workbench_controllers)/launch/dynamixel_controllers.launch" /> 
    <!-- walk controll-->
    <node name="pos_controller" pkg="jishupro_remake" type="pos_controller.py"></node>
  
  </launch>